src/components/Banner.jsx

```diff
@@ -0,0 +1,82 @@
+import React from 'react';
+import styled from 'styled-components';
+import { graphql, StaticQuery } from 'gatsby';
+import { Card, Image } from 'semantic-ui-react';
+
+class Banner extends React.Component {
+
+  renderLogos = allLogos => (
+    <Card fluid >
+      <Card.Content className="card">
+        {this.getRandom(allLogos, 4).map(logo => {
+        if(logo.node.image.file.url != null){
+          return(
+            <Image className="image" key={logo.node.id} size="small" src={logo.node.image.file.url} />
+          );
+        }
+        return null;
+      })}
+      </Card.Content>
+    </Card>
+  )
+
+  getRandom = (arr, n) => {
+    var result = new Array(n),
```

Comment:
Var is somewhat outdated. Please look into using const or let

Reply:
Didn't know, will change that.